src/transformers/models/vit_mae/modeling_tf_vit_mae.py

```diff
@@ -490,6 +492,7 @@ def call(
         return outputs
 
 
+# Copied from transformers.models.vit.modeling_tf_vit.ViTIntermediate with TFViT->TFViTMAE
```

Comment:
You acted faster than my comment !

Reply:
Yep 😅 
Fixed it! Thanks.

Did not know that a test was kept to check for the comments! Absolutely lovely.

Reply:
This fix still does not make it I think.

There are failed tests. Could you please guide me here?

Reply:
For the `check_repository_consistency`, errors like  

```python
- src/transformers/models/vit_mae/modeling_tf_vit_mae.py: copy does not match models.vit.modeling_tf_vit.TFViTSelfAttention at line 355
- src/transformers/models/vit_mae/modeling_tf_vit_mae.py: copy does not match models.vit.modeling_tf_vit.TFViTSelfOutput at line 440
```

means you put a comment stating it is a copy from another place, but actually they are different code block.
I saw gante only made a single comment in the review mentioning to add `# Copied from`, but it seems you added quite a few. Any reason?

Reply:
However, I do see in `ViTMAE`, there are quite a few `# copied from`. For TensorFlow version `TFViT` and `TFViTMAE`, I don't know if they are different and couldn't use `# copied from`.

Reply:
In the [comment](https://github.com/huggingface/transformers/pull/16255#discussion_r835516046) as I understood it Gante wanted to add the comment in all the code snippets taken from the `ViT` model.

This was my only rationale.

Reply:
I saw you add config type, great.

However, the copy statement you added were `with TFViT->TFViTMAE`, but the added type `ViTMAEConfig` vs `ViTConfig`, and therefore they are not taken into account.

I didn't verify the code blocks, but it is probably could be solved by change `with TFViT->TFViTMAE` to `with ViT->ViTMAE`.

Otherwise, another way is to use `with TFViT->TFViTMAE, ViTConfig->ViTMAEConfig`

Reply:
It would be easier if you run `make fix-copies` locally to see if the changes would work. (don't forget to stage the changed files  first)